### ISSUE TITLE
Automated tests with OmniJ broken

### ISSUE THREAD

**Issue by dexX7** [OmniLayer/OmniJ#100]
We currently have an issue with building OmniJ, see:

- https://travis-ci.org/OmniLayer/omnicore/jobs/78731353#L3946-L4020

That's the script I use:

- https://github.com/OmniLayer/omnicore/blob/omnicore-0.0.10/qa/pull-tester/omnicore-rpc-tests.sh

edit: the error message:
```
* What went wrong:
Execution failed for task ':omnij-rpc:compileGroovy'.
> com/msgilligan/bitcoinj/cli/CliCommand : Unsupported major.minor version 52.0
```

It looks like this is somehow related to the JDK version. I tried to force `oraclejdk8` and `oraclejdk7` via Travis, but this wasn't successful (although I have no way to verify, whether Travis didn't screw up).

It works fine with 79d19e5dcb1067cf7e38c34e60846e6281cb172c.

**Comment by msgilligan** [comment]
Looking into this now. It's because I added this dependency:

    compile 'com.msgilligan:bitcoinj-cli:0.0.3' 

That jar is being compiled for Java 8, but I'm going to move it back to Java 7.

**Comment by msgilligan** [comment]
I released version 0.0.4 of **bitcoinj-cli** built with `sourceCompatibility = 1.7` ([commit](https://github.com/msgilligan/bitcoinj-addons/commit/93c42744344a9652e68c8e98b29782464553e0dd)) 

I pushed ea088e7001f6e68352085228da02b3382e1051fb to **OmniJ** which uses **bitcoin-cli** 0.0.4

Hopefully this will fix the Travis build. Let me know.

**Comment by dexX7** [comment]
Thanks! It's all good now:

- https://travis-ci.org/OmniLayer/omnicore/jobs/78733476#L3999 (and following)

:)
